vigetlabs/wordpress-site-starter
# [#46] Fixing Build and updating README

# Summary

This fixes the bug with the build file location and updates the readme doc. 

## Issues

* #46

## Testing Instructions

1. Go to config.yaml and change `ENVIRONMENT` to `prod` - This only need to be change on local as on production it defaults to prod. 
2. Run `bin/build` 
3. Look at a page with the blocks on the front end. 
4. Look at the same page in the editor to make sure the styles are loading there as well. 

## Screenshots

NA
